engine/src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala

```diff
@@ -156,7 +157,10 @@ trait WdlStandardLibraryImpl extends WdlStandardLibraryFunctions {
     // This may be called multiple times with the same inputs.  Calling this twice with the same
     // parameters should yield the same return value.
     val rootDir = path.toAbsolutePath(fileSystems)
-    if (!rootDir.exists) rootDir.toFile.mkdirs()
+    Try(rootDir.toFile) match {
+      case Success(javaFile) if !javaFile.exists => javaFile.mkdirs
+      case _ =>
+    }
```

Comment:
This is just a placeholder solution in case Jose needs it soon, but I'm not sure this is the best solution